apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/GetSchemaTableTests.cs

```diff
@@ -90,6 +68,32 @@ public async Task GetSchemaTable_WithFlagEnabled_ReturnsSchema(string columnName
             }
         }
 
+        public static TheoryData<string, System.Type, SpannerDbType> SchemaTestData
```

Comment:
Sorry, I meant:

```csharp
public static TheoryData<string, System.Type, SpannerDbType> SchemaTestData { get; } =
    new TheoryData<...>
    {
        ...
    };
```

Reply:
(Let me know if you want us to merge and for me to fix that up later though.)

Reply:
Sorry, you wrote that correctly the first time. It should be fixed now (and I squashed the commits to clean up the history as well).